notebooks/tutorials/enclaves/Enclave-single-notebook-high-low-network.ipynb

```diff
@@ -13,6 +13,17 @@
     "from recordlinkage.datasets import load_febrl4"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    it_node_low = sy.Orchestra.launch(name="italy-1", node_side_type="low", local_db=True, reset=True) 
</pre><p>do we want to use high/low here or staging/production? Also, could be helpful to make it a boolean as its easier to autocomplete? is_staging?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/7916/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
For now we're using high/low on client side, we can switch to prod/staging if that's required.